### PR TITLE
fv3atm cap cleanup

### LINKED ISSUE #441: Code cleanup. Remove used code/variables. Fix minor inconsistencies.
There are a lot of unused code/variables, a lot of little inconsistencies. Time for some cleanup.
For exact code changes see PR #440.

### PR DESCRIPTION
**Pull request by DusanJovic-NOAA**
# PR Checklist

- [x] Ths PR is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR. Please consult the ufs-weather-model [wiki](https://github.com/ufs-community/ufs-weather-model/wiki/Making-code-changes-in-the-UFS-weather-model-and-its-subcomponents) if you are unsure how to do this.

- [x] This PR has been tested using a branch which is up-to-date with the top of all sub-component repositories except for those sub-components which are the subject of this PR

- [x] An Issue describing the work contained in this PR has been created either in the subcomponent(s) or in the ufs-weather-model. The Issue should be created in the repository that is most relevant to the changes in contained in the PR. The Issue and the dependent sub-component PR 
are specified below.

- [ ] Results for one or more of the regression tests change and the reasons for the changes are understood and explained below.

- [ ] New or updated input data is required by this PR. If checked, please work with the code managers to update input data sets on all platforms.

## Instructions: All subsequent sections of text should be filled in as appropriate.

The information provided below allows the code managers to understand the changes relevant to this PR, whether those changes are in the ufs-weather-model repository or in a subcomponent repository. Ufs-weather-model code managers will use the information provided to add any applicable labels, assign reviewers and place it in the Commit Queue. Once the PR is in the Commit Queue, it is the PR owner's responsiblity to keep the PR up-to-date with the develop branch of ufs-weather-model. 

## Description

FV3 nuopc cap code cleanup. Remove used code/variables. Fix minor inconsistencies.

### Issue(s) addressed

Link the issues to be closed with this PR, whether in this repository, or in another repository.
(Remember, issues must always be created before starting work on a PR branch!) 
- fixes https://github.com/NOAA-EMC/fv3atm/issues/441

## Testing

How were these changes tested? What compilers / HPCs was it tested with? Are the changes covered by regression tests? (If not, why? Do new tests need to be added?) Have regression tests and unit tests (utests) been run? On which platforms and with which compilers? (Note that unit tests can only be run on tier-1 platforms)

- [x] hera.intel
- [x] hera.gnu
- [x] orion.intel
- [x] cheyenne.intel 
- [x] cheyenne.gnu
- [x] gaea.intel 
- [x] jet.intel
- [x] wcoss_cray
- [x] wcoss_dell_p3
- [ ] opnReqTest for newly added/changed feature
- [x] CI https://github.com/ufs-community/ufs-weather-model/actions/runs/1588143860

## Dependencies

If testing this branch requires non-default branches in other repositories, list them. Those branches should have matching names (ideally).

Do PRs in upstream repositories need to be merged first?
If so add the "waiting for other repos" label and list the upstream PRs
- waiting on https://github.com/NOAA-EMC/fv3atm/pull/440

### CODE REVIEW

**Review by junwang-noaa — COMMENTED** [review]
Code looks good to me. Will approve when it's commit time.

File: tests/RegressionTests_wcoss_cray.log [hunk 1]
```diff
@@ -1,17 +1,33 @@
...
+Test 004 compile RUN_SUCCESS
+
+Test 008 compile RUN_SUCCESS
+
+Test 001 compile RUN_SUCCESS
+
+Test 003 compile RUN_SUCCESS
+
+Test 002 compile RUN_SUCCESS
+
```
Comment (DusanJovic-NOAA) [inline tests/RegressionTests_wcoss_cray.log:19]: These extra lines were added by one of the changes in rt script updates. Should be removed in one of the next PRs.

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: orion
Compiler: intel
Job: RT
Repo location: /work/noaa/nems/emc.nemspara/autort/pr/801370839/20211216103010/ufs-weather-model
Test cpld_control_p7 001 failed in run_test failed 
Test cpld_control_p7_rrtmgp 002 failed in run_test failed 
Test cpld_2threads_p7 003 failed in run_test failed 
Test cpld_decomp_p7 004 failed in run_test failed 
Test cpld_mpi_p7 005 failed in run_test failed 
Test cpld_bmark_p7 006 failed in run_test failed 
Test cpld_bmark_mpi_p7 007 failed in run_test failed 
Test cpld_control_c96_p7 008 failed in run_test failed 
Test cpld_control_c192_p7 010 failed in run_test failed 
Test cpld_control_c384_p7 012 failed in run_test failed 
Test cpld_debug_p7 014 failed in run_test failed 
Test control 015 failed in run_test failed 
Test control_decomp 016 failed in run_test failed 
Test control_2threads 017 failed in run_test failed 
Test control_fhzero 019 failed in run_test failed 
Test control_CubedSphereGrid 020 failed in run_test failed 
Test control_latlon 021 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel 022 failed in run_test failed 
Test control_c48 023 failed in run_test failed 
Test control_c192 024 failed in run_test failed 
Test control_c384 025 failed in run_test failed 
Test control_c384gdas 026 failed in run_test failed 
Test control_stochy 027 failed in run_test failed 
Test control_lndp 029 failed in run_test failed 
Test control_p7 030 failed in run_test failed 
Test control_p7_rrtmgp 031 failed in run_test failed 
Test control_decomp_p7 033 failed in run_test failed 
Test control_2threads_p7 034 failed in run_test failed 
Test regional_control 035 failed in run_test failed 
Test regional_noquilt 037 failed in run_test failed 
Test regional_2threads 038 failed in run_test failed 
Test regional_hafs 039 failed in run_test failed 
Test regional_netcdf_parallel 040 failed in run_test failed 
Test regional_RRTMGP 041 failed in run_test failed 
Test rap_control 042 failed in run_test failed 
Test rap_2threads 043 failed in run_test failed 
Test rap_sfcdiff 044 failed in run_test failed 
Test hrrr_control 046 failed in run_test failed 
Test rrfs_v1beta 047 failed in run_test failed 
Test control_rrtmgp 048 failed in run_test failed 
Test control_rrtmgp_c192 049 failed in run_test failed 
Test control_csawmg 050 failed in run_test failed 
Test control_csawmgt 051 failed in run_test failed 
Test control_flake 052 failed in run_test failed 
Test control_ras 053 failed in run_test failed 
Test control_thompson 054 failed in run_test failed 
Test control_thompson_no_aero 055 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 056 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 057 failed in run_test failed 
Test control_wam 058 failed in run_test failed 
Test control_debug 059 failed in run_test failed 
Test control_2threads_debug 060 failed in run_test failed 
Test control_CubedSphereGrid_debug 061 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel_debug 062 failed in run_test failed 
Test control_stochy_debug 063 failed in run_test failed 
Test control_lndp_debug 064 failed in run_test failed 
Test control_rrtmgp_debug 065 failed in run_test failed 
Test control_csawmg_debug 066 failed in run_test failed 
Test control_csawmgt_debug 067 failed in run_test failed 
Test control_ras_debug 068 failed in run_test failed 
Test control_diag_debug 069 failed in run_test failed 
Test control_debug_p7 070 failed in run_test failed 
Test control_thompson_debug 071 failed in run_test failed 
Test control_thompson_no_aero_debug 072 failed in run_test failed 
Test control_thompson_extdiag_debug 073 failed in run_test failed 
Test regional_debug 074 failed in run_test failed 
Test rap_control_debug 075 failed in run_test failed 
Test rap_unified_drag_suite_debug 076 failed in run_test failed 
Test rap_diag_debug 077 failed in run_test failed 
Test rap_cires_ugwp_debug 078 failed in run_test failed 
Test rap_unified_ugwp_debug 079 failed in run_test failed 
Test rap_noah_debug 080 failed in run_test failed 
Test rap_rrtmgp_debug 081 failed in run_test failed 
Test rap_lndp_debug 082 failed in run_test failed 
Test rap_sfcdiff_debug 083 failed in run_test failed 
Test rap_flake_debug 084 failed in run_test failed 
Test rap_noah_sfcdiff_cires_ugwp_debug 085 failed in run_test failed 
Test rrfs_v1beta_debug 086 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 087 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 088 failed in run_test failed 
Test hafs_regional_atm 089 failed in run_test failed 
Test hafs_regional_atm_ocn 090 failed in run_test failed 
Test hafs_regional_atm_wav 091 failed in run_test failed 
Test hafs_regional_atm_ocn_wav 092 failed in run_test failed 
Test hafs_regional_1nest_atm 093 failed in run_test failed 
Test hafs_regional_telescopic_2nests_atm 094 failed in run_test failed 
Test hafs_global_1nest_atm 095 failed in run_test failed 
Test hafs_global_multiple_4nests_atm 096 failed in run_test failed 
Test hafs_regional_docn 097 failed in run_test failed 
Test hafs_regional_docn_oisst 098 failed in run_test failed 
Test hafs_regional_datm_cdeps 099 failed in run_test failed 
Test datm_cdeps_control_cfsr 100 failed in run_test failed 
Test datm_cdeps_control_gefs 102 failed in run_test failed 
Test datm_cdeps_stochy_gefs 103 failed in run_test failed 
Test datm_cdeps_bulk_cfsr 104 failed in run_test failed 
Test datm_cdeps_bulk_gefs 105 failed in run_test failed 
Test datm_cdeps_mx025_cfsr 106 failed in run_test failed 
Test datm_cdeps_mx025_gefs 107 failed in run_test failed 
Test datm_cdeps_multiple_files_cfsr 108 failed in run_test failed 
Test datm_cdeps_debug_cfsr 109 failed in run_test failed 
Test control_atmwav 110 failed in run_test failed 
Test control_c384gdas_wav 111 failed in run_test failed 
Please make changes and add the following label back:
orion-intel-RT

**Comment by DusanJovic-NOAA** [comment]
> 

```
$ tail /work/noaa/nems/emc.nemspara/autort/pr/801370839/20211216103010/ufs-weather-model/tests/log_orion.intel/run_001_cpld_control_p7.log 
5 min. TEST 001 cpld_control_p7 is COMPLETED,  status: - jobid 3967251


Test 001 cpld_control_p7 RUN_SUCCESS
/work/noaa/nems/emc.nemspara/autort/pr/801370839/20211216103010/ufs-weather-model/tests/run_test.sh: line 240: skip_check_results: unbound variable
++ '[' 1 -eq 0 ']'
++ write_fail_test
++ [[ false == true ]]
++ echo 'cpld_control_p7 001 failed in run_test'
++ exit 1

```

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: gaea
Compiler: intel
Job: RT
Repo location: /lustre/f2/pdata/ncep/emc.nemspara/autort/pr/801370839/20211216163008/ufs-weather-model
Test cpld_control_p7 001 failed in run_test failed 
Test cpld_control_p7_rrtmgp 002 failed in run_test failed 
Test cpld_2threads_p7 003 failed in run_test failed 
Test cpld_decomp_p7 004 failed in run_test failed 
Test cpld_mpi_p7 005 failed in run_test failed 
Test cpld_bmark_p7 006 failed in run_test failed 
Test cpld_bmark_mpi_p7 007 failed in run_test failed 
Test cpld_control_c96_p7 008 failed in run_test failed 
Test cpld_control_c192_p7 010 failed in run_test failed 
Test cpld_control_c384_p7 012 failed in run_test failed 
Test cpld_debug_p7 014 failed in run_test failed 
Test control 015 failed in run_test failed 
Test control_decomp 016 failed in run_test failed 
Test control_2threads 017 failed in run_test failed 
Test control_fhzero 019 failed in run_test failed 
Test control_CubedSphereGrid 020 failed in run_test failed 
Test control_latlon 021 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel 022 failed in run_test failed 
Test control_c48 023 failed in run_test failed 
Test control_c192 024 failed in run_test failed 
Test control_c384 025 failed in run_test failed 
Test control_c384gdas 026 failed in run_test failed 
Test control_stochy 027 failed in run_test failed 
Test control_lndp 029 failed in run_test failed 
Test control_p7 030 failed in run_test failed 
Test control_p7_rrtmgp 031 failed in run_test failed 
Test control_decomp_p7 033 failed in run_test failed 
Test control_2threads_p7 034 failed in run_test failed 
Test regional_control 035 failed in run_test failed 
Test regional_noquilt 037 failed in run_test failed 
Test regional_2threads 038 failed in run_test failed 
Test regional_hafs 039 failed in run_test failed 
Test regional_netcdf_parallel 040 failed in run_test failed 
Test regional_RRTMGP 041 failed in run_test failed 
Test rap_control 042 failed in run_test failed 
Test rap_2threads 043 failed in run_test failed 
Test rap_sfcdiff 044 failed in run_test failed 
Test hrrr_control 046 failed in run_test failed 
Test rrfs_v1beta 047 failed in run_test failed 
Test control_rrtmgp 048 failed in run_test failed 
Test control_rrtmgp_c192 049 failed in run_test failed 
Test control_csawmgt 050 failed in run_test failed 
Test control_flake 051 failed in run_test failed 
Test control_ras 052 failed in run_test failed 
Test control_thompson 053 failed in run_test failed 
Test control_thompson_no_aero 054 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 055 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 056 failed in run_test failed 
Test control_wam 057 failed in run_test failed 
Test control_debug 058 failed in run_test failed 
Test control_2threads_debug 059 failed in run_test failed 
Test control_CubedSphereGrid_debug 060 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel_debug 061 failed in run_test failed 
Test control_stochy_debug 062 failed in run_test failed 
Test control_lndp_debug 063 failed in run_test failed 
Test control_rrtmgp_debug 064 failed in run_test failed 
Test control_csawmg_debug 065 failed in run_test failed 
Test control_csawmgt_debug 066 failed in run_test failed 
Test control_ras_debug 067 failed in run_test failed 
Test control_diag_debug 068 failed in run_test failed 
Test control_debug_p7 069 failed in run_test failed 
Test control_thompson_debug 070 failed in run_test failed 
Test control_thompson_no_aero_debug 071 failed in run_test failed 
Test control_thompson_extdiag_debug 072 failed in run_test failed 
Test regional_debug 073 failed in run_test failed 
Test rap_control_debug 074 failed in run_test failed 
Test rap_unified_drag_suite_debug 075 failed in run_test failed 
Test rap_diag_debug 076 failed in run_test failed 
Test rap_cires_ugwp_debug 077 failed in run_test failed 
Test rap_unified_ugwp_debug 078 failed in run_test failed 
Test rap_noah_debug 079 failed in run_test failed 
Test rap_rrtmgp_debug 080 failed in run_test failed 
Test rap_lndp_debug 081 failed in run_test failed 
Test rap_sfcdiff_debug 082 failed in run_test failed 
Test rap_flake_debug 083 failed in run_test failed 
Test rap_noah_sfcdiff_cires_ugwp_debug 084 failed in run_test failed 
Test rrfs_v1beta_debug 085 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 086 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 087 failed in run_test failed 
Test hafs_regional_atm 088 failed in run_test failed 
Test hafs_regional_atm_ocn 089 failed in run_test failed 
Test hafs_regional_atm_wav 090 failed in run_test failed 
Test hafs_regional_atm_ocn_wav 091 failed in run_test failed 
Test hafs_regional_1nest_atm 092 failed in run_test failed 
Test hafs_regional_telescopic_2nests_atm 093 failed in run_test failed 
Test hafs_global_1nest_atm 094 failed in run_test failed 
Test hafs_global_multiple_4nests_atm 095 failed in run_test failed 
Test hafs_regional_docn 096 failed in run_test failed 
Test hafs_regional_docn_oisst 097 failed in run_test failed 
Test hafs_regional_datm_cdeps 098 failed in run_test failed 
Test datm_cdeps_control_cfsr 099 failed in run_test failed 
Test datm_cdeps_control_gefs 101 failed in run_test failed 
Test datm_cdeps_stochy_gefs 102 failed in run_test failed 
Test datm_cdeps_bulk_cfsr 103 failed in run_test failed 
Test datm_cdeps_bulk_gefs 104 failed in run_test failed 
Test datm_cdeps_mx025_cfsr 105 failed in run_test failed 
Test datm_cdeps_mx025_gefs 106 failed in run_test failed 
Test datm_cdeps_multiple_files_cfsr 107 failed in run_test failed 
Test datm_cdeps_debug_cfsr 108 failed in run_test failed 
Test control_atmwav 109 failed in run_test failed 
Test control_c384gdas_wav 110 failed in run_test failed 
Please make changes and add the following label back:
gaea-intel-RT

**Comment by BrianCurtis-NOAA** [comment]
***Automated RT Failure Notification***
Machine: jet
Compiler: intel
Job: RT
Repo location: /lfs4/HFIP/h-nems/emc.nemspara/autort/pr/801370839/20211216163012/ufs-weather-model
Test cpld_control_p7 001 failed in run_test failed 
Test cpld_control_p7_rrtmgp 002 failed in run_test failed 
Test cpld_2threads_p7 003 failed in run_test failed 
Test cpld_mpi_p7 004 failed in run_test failed 
Test cpld_bmark_p7 005 failed in run_test failed 
Test cpld_bmark_mpi_p7 006 failed in run_test failed 
Test cpld_control_c96_p7 007 failed in run_test failed 
Test cpld_control_c192_p7 009 failed in run_test failed 
Test cpld_control_c384_p7 011 failed in run_test failed 
Test cpld_debug_p7 013 failed in run_test failed 
Test control 014 failed in run_test failed 
Test control_decomp 015 failed in run_test failed 
Test control_2threads 016 failed in run_test failed 
Test control_fhzero 018 failed in run_test failed 
Test control_CubedSphereGrid 019 failed in run_test failed 
Test control_latlon 020 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel 021 failed in run_test failed 
Test control_c48 022 failed in run_test failed 
Test control_c192 023 failed in run_test failed 
Test control_c384 024 failed in run_test failed 
Test control_c384gdas 025 failed in run_test failed 
Test control_stochy 026 failed in run_test failed 
Test control_lndp 028 failed in run_test failed 
Test control_p7 029 failed in run_test failed 
Test control_p7_rrtmgp 030 failed in run_test failed 
Test control_decomp_p7 032 failed in run_test failed 
Test control_2threads_p7 033 failed in run_test failed 
Test regional_control 034 failed in run_test failed 
Test regional_noquilt 036 failed in run_test failed 
Test regional_hafs 037 failed in run_test failed 
Test regional_netcdf_parallel 038 failed in run_test failed 
Test regional_RRTMGP 039 failed in run_test failed 
Test rap_control 040 failed in run_test failed 
Test rap_sfcdiff 041 failed in run_test failed 
Test hrrr_control 043 failed in run_test failed 
Test rrfs_v1beta 044 failed in run_test failed 
Test control_rrtmgp 045 failed in run_test failed 
Test control_rrtmgp_c192 046 failed in run_test failed 
Test control_csawmg 047 failed in run_test failed 
Test control_csawmgt 048 failed in run_test failed 
Test control_flake 049 failed in run_test failed 
Test control_ras 050 failed in run_test failed 
Test control_thompson 051 failed in run_test failed 
Test control_thompson_no_aero 052 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson 053 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson 054 failed in run_test failed 
Test control_wam 055 failed in run_test failed 
Test control_debug 056 failed in run_test failed 
Test control_2threads_debug 057 failed in run_test failed 
Test control_CubedSphereGrid_debug 058 failed in run_test failed 
Test control_wrtGauss_netcdf_parallel_debug 059 failed in run_test failed 
Test control_stochy_debug 060 failed in run_test failed 
Test control_lndp_debug 061 failed in run_test failed 
Test control_rrtmgp_debug 062 failed in run_test failed 
Test control_csawmg_debug 063 failed in run_test failed 
Test control_csawmgt_debug 064 failed in run_test failed 
Test control_ras_debug 065 failed in run_test failed 
Test control_diag_debug 066 failed in run_test failed 
Test control_debug_p7 067 failed in run_test failed 
Test control_thompson_debug 068 failed in run_test failed 
Test control_thompson_no_aero_debug 069 failed in run_test failed 
Test control_thompson_extdiag_debug 070 failed in run_test failed 
Test regional_debug 071 failed in run_test failed 
Test rap_control_debug 072 failed in run_test failed 
Test rap_unified_drag_suite_debug 073 failed in run_test failed 
Test rap_diag_debug 074 failed in run_test failed 
Test rap_cires_ugwp_debug 075 failed in run_test failed 
Test rap_unified_ugwp_debug 076 failed in run_test failed 
Test rap_noah_debug 077 failed in run_test failed 
Test rap_rrtmgp_debug 078 failed in run_test failed 
Test rap_lndp_debug 079 failed in run_test failed 
Test rap_sfcdiff_debug 080 failed in run_test failed 
Test rap_flake_debug 081 failed in run_test failed 
Test rap_noah_sfcdiff_cires_ugwp_debug 082 failed in run_test failed 
Test rrfs_v1beta_debug 083 failed in run_test failed 
Test fv3_HAFS_v0_hwrf_thompson_debug 084 failed in run_test failed 
Test fv3_esg_HAFS_v0_hwrf_thompson_debug 085 failed in run_test failed 
Test hafs_regional_atm 086 failed in run_test failed 
Test hafs_regional_atm_ocn 087 failed in run_test failed 
Test hafs_regional_atm_wav 088 failed in run_test failed 
Test hafs_regional_atm_ocn_wav 089 failed in run_test failed 
Test hafs_regional_docn 090 failed 
Test hafs_regional_docn 090 failed in run_test failed 
Test hafs_regional_docn_oisst 091 failed in run_test failed 
Test hafs_regional_datm_cdeps 092 failed in run_test failed 
Test datm_cdeps_control_cfsr 093 failed in run_test failed 
Test datm_cdeps_control_gefs 095 failed in run_test failed 
Test datm_cdeps_stochy_gefs 096 failed in run_test failed 
Test datm_cdeps_bulk_cfsr 097 failed in run_test failed 
Test datm_cdeps_bulk_gefs 098 failed in run_test failed 
Test datm_cdeps_mx025_cfsr 099 failed in run_test failed 
Test datm_cdeps_mx025_gefs 100 failed in run_test failed 
Test datm_cdeps_multiple_files_cfsr 101 failed in run_test failed 
Test datm_cdeps_debug_cfsr 102 failed in run_test failed 
Test control_atmwav 103 failed in run_test failed 
Please make changes and add the following label back:
jet-intel-RT

**Review by junwang-noaa — APPROVED** [review]

**Review by DeniseWorthen — APPROVED** [review]

**Review by climbfuji — APPROVED** [review]
Should I merge?

**Pull request merged**